src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -207,7 +209,20 @@ public static void RoundtripString()
             Assert.Equal("12-34-56-78-9A", BitConverter.ToString(bytes));
             Assert.Equal("56-78-9A", BitConverter.ToString(bytes, 2));
             Assert.Equal("56", BitConverter.ToString(bytes, 2, 1));
-            Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
+
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[0]));
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[3], 1, 0));
```

Comment:
> When tests are open-source, they become documentation of a sort in themselves

Why is that the case more-so than the implementation being open-source?

 > A comment of "not promised by spec" or "consider impact before changing this behaviour" would remove that downside

But I believe it adds its own.  To me, that then implies that if there isn't such a comment, that these comments don't apply, yet there are such tests being done throughout the repo.